content/docs/rfcs/31/README.md

```diff
@@ -0,0 +1,21 @@
+---
+slug: 31
+title: 31/WAKU2-TRANSPORT
+name: Waku v2 Transport Specification
+status: draft
+editor: Reeshav Khan <reeshav@status.im>
+contributors:
+---
+
+`31/WAKU2-TRANSPORT` describes the specification for Protocols supported by waku. While Waku strives to be transport agonistic, however any underlying transport layer MUST be IETF standardized transport protocol, this document underlines the baseline for interoperability between different waku clients.
+
+# Transport Protocol Specification 
+
+Waku client implementations MUST support the TCP transport, and it MUST be enabled for both dialing and listening even if other transports are available. During negotiation upgrade protocol MAY be chosen over TCP. 
+The underlying transports SHOULD support websocket and secure websockets for bidirectional communication streams and browser applications. 
```

Comment:
I'd say SHOULD support secure websockets and MAY support insecure websockets, as required by the application.